src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -103,6 +103,95 @@ const PIAFormPage = () => {
       ? 'view'
       : 'edit';
 
+  const hasFormChanged = useCallback(() => {
+    return !deepEqual(stalePia, pia, ['updatedAt', 'saveId']);
+  }, [pia, stalePia]);
+
+  const sendSnowplowStatusChangeCall = (completedStatus?: boolean) => {
+    window.snowplow('trackSelfDescribingEvent', {
+      schema: 'iglu:ca.bc.gov.cirmo/dpia_progress/jsonschema/1-0-0',
+      data: {
+        id: pia?.id,
+        title: pia?.title,
+        state: completedStatus ? `${pia?.status}-COMPLETED` : pia?.status,
+        ministry: pia?.ministry,
+        branch: pia?.branch,
+      },
```

Comment:
let us track delegated pia and full pia too. What do you think ?

Reply:
This is just the call that Dan gave us in the ticket